com.unity.visualeffectgraph/Documentation~/Reference-Attributes.md

```diff
@@ -79,7 +79,9 @@ Setting a Lifetime attribute to a particle in an Initialize Context, will implic
 
 > Automatic particle reaping can be disabled by selecting the Update Context, then setting the Reap Particles value to **False**.
 
-> **Immortal Particles:** Particles with no lifetime will be considered as immortal. You can still kill them explicitly by setting their `alive` attribute to `false`.
+> **Immortal Particles:** Particles with no lifetime are considered immortal. You can still kill them explicitly by setting their `alive` attribute to `false`.
+
+**Note**: If you set the alive attribute to false in an output context, the particle will be culled from rendering but not killed.
```

Comment:
I know API docs read very differently but general style rules still apply, so let's get rid of that passive voice :) (also attricute names should be ``'d to be in monospace:

**Note**: When you set the `alive` attribute to false in an output context, Unity culls the particle from rendering but doesn't kill it.